pipettes/firmware/main.cpp

```diff
@@ -9,12 +9,19 @@
 
 // clang-format on
 
+#include "common/core/freertos_timer.hpp"
 #include "common/firmware/clocking.h"
 
+static void callback() {}
+
 auto main() -> int {
     HardwareInit();
     RCC_Peripheral_Clock_Select();
     MX_ICACHE_Init();
 
+    // Dummy timer code, should be removed once SW timers are implemented
+    auto timer = freertos_timer::FreeRTOSTimer("test timer", callback);
```

Comment:
do we want to commit this

Reply:
Nah was leaving it there till someone reviewed it. I was planning to take it out before merging.